src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,6 +6,12 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_COMPANY = new Prefix("c/");
+    public static final Prefix PREFIX_CONTACT = new Prefix("p/");
+    public static final Prefix PREFIX_POSITION = new Prefix("p/");
+    public static final Prefix PREFIX_DATE = new Prefix("d/");
+
+    /* deprecated */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
```

Comment:
Since we are using email and tag, perhaps move it to the top, out of this 'deprecated' comment?

Reply:
Fixed 👍